demo/corpus/logic/src/vectorstore/management.ts

```diff
@@ -10,11 +10,15 @@ import { ENV } from '../env';
  * - Add pgvector extension
  * - Create table(s)
  */
-export async function initializeVectorStore(truncate: boolean = false) {
+export async function initializeVectorStore(embeddingTableName: string, vectorSize: number, truncate: boolean = false) {
```

Comment:
nitpick: we could create a class (e.g. `VectorStoreManager`) and pull these methods in it

Reply:
I'd defer this to next iteration when we want to refactor the vector store.